impress-org/givewp
# Feature: Generate Tickets for Donation

Resolves [GIVE-370](https://stellarwp.atlassian.net/browse/GIVE-370)

## Description



This PR generates tickets for each ticket type purchased during the donation process.

Note: This also resolves a small bug with the `created_at` and `updated_at` timestamps when creating a Ticket.

## Testing Instructions



Select a number of tickets when making a donation.

The generated tickets should be added to the `give_event_tickets` table.

[GIVE-370]: https://stellarwp.atlassian.net/browse/GIVE-370?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
